changes/issue1588.yaml

```diff
@@ -0,0 +1,2 @@
+fix:
+  - Kubernetes agent resource manager is more strict about what resources it manages - [#2641](https://github.com/PrefectHQ/prefect/pull/2641)
```

Comment:
Could you also add to this that the identifier labels on the environments changed?

Reply:
Should I note that as a breaking change? It likely won't affect most users, but it may affect some.

Reply:
Yeah I'm cool with that 👍